tests/inputFields.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { test, expect } from '@playwright/test';
+import exp from 'constants';
+
+test.beforeEach(async({page}) => {
+    await page.goto('https://petclinic.bondaracademy.com/welcome')
+})
+
+test.describe('Pet types', () => {
+    test.beforeEach(async({page}) => {
+        await page.getByRole('link', {name: "Pet Types"}).click() //1. Select Pet Types menu
+    })
+
+    test('Update pet type', async({page}) => {
+        const petTypesTableName = page.getByRole('heading', {name: "Pet Types"})
+        await expect(petTypesTableName).toHaveText('Pet Types')  //2. assertion of Pet Types text displayed
+
+        await page.getByRole('button', {name: "Edit"}).first().click()  //3. click on edit button for cat
+        await expect(page.getByRole('heading', {name: "Edit Pet Type"})).toBeVisible() //4. assertion of Edit Pet Type header
+
+        const editPetTypeName = page.locator('#name')
+        await editPetTypeName.click()
+        await editPetTypeName.clear()
+        await editPetTypeName.fill('rabbit') //5. change pet name to rabbit and click update
+        const updatedPetTypeNameValue = await editPetTypeName.inputValue()
+        expect(updatedPetTypeNameValue).toEqual('rabbit')
```

Comment:
lines 24 and 25 can be replaced with a single line locator assertion, since you don't use `updatedPetTypeNameValue` anywhere else in the code

Reply:
No you can't. This is incorrect syntax first of all.
Please check the lesson about the assertions one more time, so you ll have a clear understanding between generic assertions and locator assertions. It's important to understand.

Reply:
1. Two lines here two lines there.. eventually results in the code that is harder to read and maintain on real projects. So always need to optimize the code when possible.
2. Locaor assertions has own timeout, while regular methods like inputValue() will wait up to the test timeout. In case of failure condition it unnecessary increases the execution of the test. So it's always preferable to use locator assertions instead of generic assertions.

Reply:
Ok, I will work on this fix. I think it will take me some time to sort out.

Reply:
I think I figured it out. Applied all updates you have suggested from the PR. This branch should be current with all changes made.